FirebaseMessaging.podspec

```diff
@@ -35,15 +35,23 @@ device, and it is completely free.
 
   base_dir = "FirebaseMessaging/"
   s.source_files = [
-    base_dir + 'Sources/**/*.[mh]',
+    base_dir + 'Sources/**/*',
+    base_dir + 'Sources/Protogen/nanopb/*.h',
     'Interop/Analytics/Public/*.h',
     'FirebaseCore/Sources/Private/*.h',
     'FirebaseInstallations/Source/Library/Private/*.h',
+    'GoogleDataTransport/GDTCORLibrary/Public/GoogleDataTransport/*.h',
```

Comment:
GoogleDataTransport is no longer available in this repo.  Private headers should not be accessed and public headers should be accessed like `#import <GoogleDataTransport/Public.h>`